runtime/appruntime/metrics/aws/cloudwatch.go

```diff
@@ -156,3 +158,19 @@ func (x *Exporter) getClient() *cloudwatch.Client {
 	}
 	return x.client
 }
+
+func getSysMetrics(now time.Time) []types.MetricDatum {
+	sysMetrics := system.ReadSysMetrics()
+	return []types.MetricDatum{
+		{
+			MetricName: aws.String("memory_usage_bytes"),
+			Timestamp:  aws.Time(now),
+			Value:      aws.Float64(float64(sysMetrics.MemoryUsageBytes)),
+		},
+		{
+			MetricName: aws.String("num_go_routines"),
+			Timestamp:  aws.Time(now),
+			Value:      aws.Float64(float64(sysMetrics.NumGoRoutines)),
```

Comment:
I thought all encore inbuilt metrics started with `e_` so that users own metrics wouldn't overlap with ours?

Reply:
Good catch!